hoomd/hpmc/integrate.py

```diff
@@ -1430,42 +1445,47 @@ def type_shapes(self):
 
 
 class ConvexSpheropolyhedronUnion(_HPMCIntegrator):
-    R""" HPMC integration for unions of convex polyhedra (3D).
-
-    Args:
-        seed (int): Random number seed.
-        d (float): Maximum move displacement, Scalar to set for all types, or a
-          dict containing {type:size} to set by type.
-        a (float): Maximum rotation move, Scalar to set for all types, or a dict
-          containing {type:size} to set by type.
-        move_ratio (float): Ratio of translation moves to rotation moves.
-        nselect (int): The number of trial moves to perform in each cell.
-        deterministic (bool): Make HPMC integration deterministic on the GPU
-
-    .. versionadded:: 2.2
-
-    Convex polyhedron union parameters:
-
-    * *shapes* (**required**) - list of polyhedron objects to be included in
-      union
-    * *positions* (**required**) - list of centers of constituent polyhedra in
-      particle coordinates.
-    * *orientations* (**required**) - list of orientations of constituent
-      polyhedra.
-    * *overlap* (**default: 1 for all particles**) - only check overlap between
-      constituent particles for which *overlap [i] & overlap[j]* is !=0, where
-      '&' is the bitwise AND operator.
-    * *sweep_radii* (**default: 0 for all particle**) - radii of spheres
-      sweeping out each constituent polyhedron
-
-        * .. versionadded:: 2.4
-
-    * *ignore_statistics* (**default: False**) - set to True to disable ignore
-      for statistics tracking.
-    * *capacity* (**default: 4**) - set to the maximum number of particles per
-      leaf node for better performance
-
-        * .. versionadded:: 2.2
+    R""" Hard particle Monte Carlo integration method for unions of convex
+    polyhedra (3D).
+
+    ConvexSpheropolyhedronUnion parameters:
+
+    shape (particle type, dict): defines the shape of the object.
+
+        Keys:
+            * *shapes* (**required**) - list of polyhedron objects to be
+              included in union
+            * *positions* (**required**) - list of centers of constituent
+              polyhedra in particle coordinates.
+            * *orientations* (**required**) - list of orientations of
+              constituent polyhedra.
+            * *overlap* (**default: 1**) - only check overlap
```

Comment:
default's meaning in this context is not described.

Reply:
fixed.